src/screens/select_track_screen/track_box.cpp

```diff
@@ -141,18 +140,23 @@ TrackBox::TrackBox(
 
 
 
-
-    // init the part displaying the midi output used
     playChoiceLabel.setFont(font);
     playChoiceLabel.setCharacterSize(CHARACTER_SIZE);
-    playChoiceLabel.setString(playChoice);
+    //playChoiceLabel.setString(playChoice); // String will be set when the track will have been chosen
```

Comment:
if you comment it, can you just remove the line :)

Reply:
I made it a real comment. Is this OK for you ?

Reply:
fair enough :)